examples/ticketing-connector/README.me

```diff
@@ -0,0 +1,39 @@
+## Ticketing Example
+
+The `ticketing-connector` example demonstrates how to use Synqly's Ticketing Connector to interact with a ticketing system. This example uses Jira, but could be adapted to use any of the ticketing systems supported by Synqly.
+
+The `ticketing-connector` example performs the following actions (in order):
+
+- Creates an account in your Synqly Organization
+- Creates an `Integration` in this Account (using Jira as the target Ticketing Provider)
+- Retrieves the list of projects
+- Retrieves the list of remote fields
+- Creates a ticket
+- Creates an attachment in the ticket
+- Downloads an attachment
+- Deletes an attachment in the ticket
+- Creates a comment in the ticket
+- Retrieves all comments in the ticket
+- Deletes a comment in the ticket
+- Updates a ticket
+- Cleans up the created tenant
```

Comment:
This is very nice, sets up a story of what we can learn from this example. Maybe we can reword it like that as well?

```suggestion
Following this comprehensive example, you will have learned how to:

1. Create an Account in your Synqly Organization
2. Create an Integration in the Account, using Jira as the Ticketing Provider
3. Query the list of projects
4. Query the list of remote fields
5. Create a ticket
6. Attach a file to the ticket
7. Download the attachment
8. Delete the attachment
9. Comment on a ticket
10. List all comments to a ticket
11. Delete a comment from the ticket
12. Updates the ticket
13. Remove the test Account
```